model/cluster.go

```diff
@@ -26,6 +26,7 @@ type Cluster struct {
 	LockAcquiredBy          *string
 	LockAcquiredAt          int64
 	Networking              string
+	VPC                     string
```

Comment:
So, I believe we shouldn't set the VPC value here in the cluster object as well as the KopsMetadata. We should only need it in the metadata where the other relevant values are.

I realize that Networking is set in the cluster object as well from your previous PR, but that was a bit of an oversight on my part. It should also probably just be set a single time in the KopsMetadata. Not a big deal, but we should avoid adding duplicates of the same values as it just leads to additional complexity and more change to have the values get out of sync.

Reply:
As per your sugestion , I've removed the VPC field from the cluster object but idea was to represents it in the output of `cloud cluster list` e.g

` "CustomInstanceGroups": {},
     "Networking": "weave",
      "VPC": "vpc-094c270d675886cbb"`